src/main/java/io/cdap/plugin/salesforce/SalesforceBulkUtil.java

```diff
@@ -163,13 +166,28 @@ public static void awaitCompletion(BulkConnection bulkConnection, JobInfo job,
 
           for (BatchInfo b : statusList) {
             if (b.getState() == BatchStateEnum.Failed) {
-              throw new BulkAPIBatchException("Batch failed", b);
+              if (ignoreFailures) {
+                LOG.error(String.format("Batch failed with error: '%s'. BatchId='%s'",
```

Comment:
Please change the message to also include that this error will be ignored and pipeline will continue.